updates to clarify (#134)
lsst/dp0-2_lsst_io: data-access-analysis-tools/adql-recipes.rst

Individual objects
==================

In the above example, a single object was desired, and a statement like ``WHERE objectId=1486`` was used.
However, if more than a few single objects are desired and their ``objectId`` are known, 

> then you can use ``WHERE objectId IN ()`` to return results for all objects in a single query.

Reviewer: ```suggestion
then you can use ``WHERE objectId IN (1487, 1488, 1489)``, for example, to return results for all of the objects in a single query.
```

> Below, a list of just 12 ``objectId`` is put in a string called ``my_list``, formatted as a python tuple (with round brackets). 

Reviewer: ```suggestion
Below, a list of 12 ``objectId`` values is put in a string called ``my_list``. 
```
